### PR TITLE
refactor(build): remove bower, update workflow

### PR DESCRIPTION
**Pull request by yhuard**
And make the live reloading better!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,34 +10,37 @@
   "private": true,
   "license": "EUPL-1.1",
   "scripts": {
-    "start": "npm run dev",
-    "postinstall": "bower install && node ./scripts/tasks/vendors.js",
+    "build:scripts": "rollup --config src/framework/rollup.config.js",
```
Comment (kalinchernev) [inline package.json]: out of pure curiosity, how is the rollup local module treated as a global here?

File: package.json [hunk 2]
```diff
@@ -10,34 +10,37 @@
...
+    "build:styleguide": "gitbook build . ./build/styleguide",
+    "build:rest": "node ./scripts/build.js",
+    "build": "npm-run-all --parallel build:*",
+    "clean": "rimraf .tmp build dist docs",
```
Comment (kalinchernev) [inline package.json]: beauty!

File: package.json [hunk 3]
```diff
@@ -10,34 +10,37 @@
...
+    "dist": "NODE_ENV=production npm run clean && node npm run build && node ./scripts/dist.js",
```
Comment (kalinchernev) [inline package.json]: `&& node npm run build` => `&& npm run build`

**Comment by kalinchernev** [comment]
Just an idea that comes to mind: maybe the lint task could be also ran on change?
Similar to the [common web app style for example](https://github.com/yeoman/generator-webapp/blob/master/app/templates/gulpfile.js#L163).

File: src/framework/rollup.config.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  moduleName: 'Farnet',
+  exports: 'named',
+  plugins: [
+    resolve({
+      jsnext: true,
+      main: true,
+      browser: true,
+    }),
+    commonjs(),
+    eslint(),
```
Comment (yhuard) [inline src/framework/rollup.config.js]: @kalinchernev this line actually triggers eslint during the build 😉

Comment (yhuard) [inline src/framework/rollup.config.js]: I could do the same with `stylelint`

Comment (kalinchernev) [inline src/framework/rollup.config.js]: Yes, please :)

**Comment by yhuard** [comment]
Done!

**Pull request merged**